### PR TITLE
[ONNX] Fix roi_align ONNX export

### PR DESCRIPTION
**Pull request by KsenijaS**
Allow export of roi_align when sampling rate is negative

### CODE REVIEW

**Comment by fmassa** [comment]
@KsenijaS is the PR ready for review?

**Comment by KsenijaS** [comment]
> @KsenijaS is the PR ready for review?

@fmassa  Yes, PR is ready for a review. It would be also good that some one from my team review it as well.
@neginraoof @BowenBao

File: torchvision/ops/_register_onnx_ops.py [hunk 1]
```diff
@@ -29,6 +29,11 @@ def roi_align(g, input, rois, spatial_scale, pooled_height, pooled_width, sampli
                           " ONNX forces ROIs to be 1x1 or larger.")
             scale = torch.tensor(0.5 / spatial_scale).to(dtype=torch.float)
             rois = g.op("Sub", rois, scale)
+
+        # ONNX doesn't support negative sampling_ratio
+        if sampling_ratio < 0:
+            warnings.warn("ONNX doesn't support negative sampling ratio, therefore is is set to 0 in order to be exported.)
```
Comment (fmassa) [inline torchvision/ops/_register_onnx_ops.py:35]: This is a syntax error, as the `"` is missing in the end

**Review by fmassa — CHANGES_REQUESTED** [review]
The changes look generally good to me, but there is a syntax error in the warning message that needs to be fixed before merging

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

**Pull request merged**